config-templates/redhat/odf-local/4.14/custom-parameters.json

```diff
@@ -30,6 +30,14 @@
     "required": "true",
     "mutable": "true"
   },
+  {
+    "description": "A comma separated list of the workerpools names where you want to deploy ODF. Either specify workerpools or workernodes to deploy ODF. if not specified any of them, ODF will deploy on all nodes",
```

Comment:
```suggestion
    "description": "Specify a comma separated list of either the names or the IDs of the worker pools you want to use for ODF. If you want to use all the worker nodes in your cluster for ODF, then don't specify this option. ",
```

Reply:
@derekpoindexter Customer can select any of the below options
1. workerpool/list of workerpools -> ODF will install on all nodes under specified workerpools
2. list of worker nodes - ODF will install on specified workernodes
3. not specified any workerpool/workernode - ODF will install on all nodes. 

The second condition is not added in above description. I think we have to specify those also to understand better